### DISCUSSION TITLE
Desktop App for Win 32bit

### DISCUSSION THREAD

**Discussion by Jryxy** [rizwansoaib/whatsapp-monitor#112] [Q&A]
It will be possible to create the app for x86 / 32bit Windows system? I can’t install the EXE file on my PC

**Comment by rizwansoaib** [comment]
I have 64bit machine I had tried.

You can run from source and compile on your machine to generate exe for 32bit

**Reply by Jryxy** [reply]
Thanks for your reply. Can you tell me how in detail? I’m kinda new to this.
